components/Banner/src/MDCBannerView.h

```diff
@@ -16,6 +16,14 @@
 
 #import "MaterialButtons.h"
 
+typedef NS_ENUM(NSInteger, MDCBannerViewLayoutMode) {
```

Comment:
Please add documentation for this enumeration.

Reply:
Addressed in b5d725fdd272ec093f273a732c8948206c06f345.

Reply:
Please make that change in this PR (or if you need to refactor it, please do the refactoring in a separate PR).